test/TitleField_test.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from "chai";
+
+import React from "react";
+import { renderIntoDocument } from "react-addons-test-utils";
+import { findDOMNode } from "react-dom";
+
+import TitleField from "../src/components/fields/TitleField";
+import { createSandbox } from "./test_utils";
+
+describe("TitleField", () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("should return a legend", () => {
+    const props = {
+      title: "Field title",
+      required: true
+    };
+    const comp = renderIntoDocument(TitleField(props));
```

Comment:
I tried this at first, but it seemed that because TitleField is a function component, not a class, it won't work. It return just null.